libs/embed/src/embed.ts

```diff
@@ -6,6 +6,7 @@ import iFrameResize from 'iframe-resizer';
 import * as EventTypes from './shared/eventTypes';
 import { UnmountedError, DomainVerificationError } from './shared/errors';
 import { IFRAME_URL } from './shared/resources';
+import { INovuThemeProvider } from '@novu/notification-center/dist/esm/types/store/novu-theme-provider.context';
```

Comment:
```suggestion
import { INovuTheme } from '@novu/notification-center';
```

the same can be applied here, the theme provider is an internal interface so better to use this one 🥳

Reply:
See my answer above 😅